cmake/OpenCVUtils.cmake

```diff
@@ -1754,7 +1754,7 @@ endfunction()
 
 macro(ocv_add_testdata basedir dest_subdir)
   if(BUILD_TESTS)
-    if(NOT CMAKE_CROSSCOMPILING AND NOT INSTALL_TESTS)
+    if(NOT INSTALL_TESTS)
```

Comment:
In general there no sense to "copy" test data into build directory during **cross-compilation**. Usually we can't run tests from build directory anyway on host platform.
We need to "install" and upload tests to target platform instead.

The same is above. `make test` doesn't work in general for **cross-compilation** so it is not supported.